### ISSUE TITLE
[bug]: Parsing error when props as parameter

### ISSUE THREAD

**Issue by baiwusanyu-c** [vue-vine/vue-vine#38]
# Bug reproduce sample
```
export function App(props: { color: string }) {
  return vine`
    <div>{{color}}</div>
  `
}
```
# Compile Result
![image](https://github.com/vue-vine/vue-vine/assets/32354856/4e726050-5cf4-420c-a5a2-2d7162fe0dba)

**Comment by baiwusanyu-c** [comment]
relate: #36

**Comment by ShenQingchuan** [comment]
Also closed by a13feb0
